platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/internal/cc/impl/ConfigurationCacheProblemReportingIntegrationTest.groovy

```diff
@@ -812,92 +744,13 @@ class ConfigurationCacheProblemReportingIntegrationTest extends AbstractConfigur
         then:
         executed(':problems', ':moreProblems', ':all')
         configurationCache.assertStateLoaded()
-        outputContains("Configuration cache entry reused with 4 problems.")
+        outputContains("Configuration cache entry reused with 3 problems.")
         problems.assertFailureHasProblems(failure) {
-            withProblem("Build file 'build.gradle': line 8: invocation of 'Task.project' at execution time is unsupported.")
             withProblem("Task `:moreProblems` of type `BrokenTask`: cannot deserialize object of type 'org.gradle.api.Project' as these are not supported with the configuration cache.")
+            withProblem("Task `:moreProblems` of type `BrokenTask`: cannot deserialize object of type 'org.gradle.api.artifacts.ConfigurationContainer' as these are not supported with the configuration cache.")
             withProblem("Task `:problems` of type `BrokenTask`: cannot deserialize object of type 'org.gradle.api.Project' as these are not supported with the configuration cache.")
-            totalProblemsCount = 4
-            problemsWithStackTraceCount = 2
-        }
-    }
-
-    def "reports #invocation access during execution"() {
```

Comment:
ℹ️ replaced by the new tests in `ConfigurationCacheTaskExecutionIntegrationTest`